templates/definition/charger/ac-elwa-2.yaml

```
@@ -0,0 +1,58 @@
+template: ac-elwa-2
+products:
+  - brand: my-PV
+    description:
+      generic: AC ELWA 2
+params:
+  - name: modbus
+    choice: ["tcpip"]
+render: |
+  type: custom
+  enable:
+    source: modbus
+    {{- include "modbus" . | indent 2 }}
+    register:
+      address: 1081 # or 1000
```

```suggestion
      address: 1000
```

wegen Non-Flash-Write